Move agreements between orgnos and MSNs (#203)
vippsas/vipps-recurring-api: vipps-recurring-api-faq.md

* Paper invoice: Include a QR code with unique URL for the customer, same as
  above.

## How can I move agreements between organizations and sale units?

Merchants sometimes need to move customer agreements from one organization
number ("orgno") to another, or from one sale unit
(merchant serial number, "MSN") to another.

First of all: A business can not "change organization number". An organization
number uniquely identifies a juridical unit, and can not be changed.

1. If the merchant wants to move agreements from
   one MSN to another,

>    aqnd both MSNs are under the same orgno,

Reviewer: ```suggestion
   and both MSNs are under the same orgno,
```

* Paper invoice: Include a QR code with unique URL for the customer, same as
  above.

## How can I move agreements between organizations and sale units?

Merchants sometimes need to move customer agreements from one organization
number ("orgno") to another, or from one sale unit
(merchant serial number, "MSN") to another.

First of all: A merchant can not "change organization number". An organization

> number uniquely identifies a juridical unit, and can not be changed.

Reviewer 2: ```suggestion
number uniquely identifies a juridical unit and cannot be changed.
```

Reviewer 2: nitpicking :-D

Reviewer 2: you could use "cannot" instead of "can not" everywhere, if you want.

   automatically move the agreements without the user's consent to enter
   agreements with the new orgno.
   The users may need to enter new agreements with the new orgno.
4. If the merchant wants to move agreements between two independent orgnos
   (not owned by a parent orgno),
   splits or fissions out a new orgno,
   is merged with another orgno,
   is acquired by another orgno,
   etc:
   Vipps may *_not_* be able to help.
   The users have entered agreements with an orgno, and Vipps can not
   automatically move the agreements without the user's consent to enter
   agreements with the new orgno.
   The users may need to enter new agreements with the new orgno.

The general process is:
- Log in on
  [portal.vipps.no](https://portal.vipps.no)
  and enter a new agreement with Vipps for the new orgno.
- Order "Vipps Faste betalinger" for the new orno. on
  [portal.vipps.no](https://portal.vipps.no).
- Inform all existing customers of the new orgno.
- Send a confirmation to avtale@vipps.no that all customers have been informed,
  including a copy of the information sent.
- Contact Vipps (your KAM or partner manager) to plan the switch, and

>   the date and time for moving the agreements.

Reviewer 3: I think maybe the order should be different here, since they should try to inform their users about the time of the migration, which necessitates that they have already organized the migration with vipps?

Author: Hmm, good point. Will tweak.

> If an agreement is moved, it will get a new `agreementId`, to prevent any

Reviewer 3: ```suggestion
Agreements are not actually moved. Your new saleunit will get new agreements identical to the old ones, but with new agreementIds. The old agreements will be stopped. This is done to prevent any
```

> confusion around charges, refunds, etc between the two MSNs.

Reviewer 3: ```suggestion
confusion around charges, refunds, etc between the two MSNs. Charges done by the old MSN are in the old agreement, while the new MSN uses only the new agreement
```
